openpype/hosts/blender/plugins/load/load_abc.py

```diff
@@ -60,18 +60,35 @@ def _process(self, libpath, asset_group, group_name):
 
         imported = lib.get_selection()
 
-        # Children must be linked before parents,
-        # otherwise the hierarchy will break
+        empties = [obj for obj in imported if obj.type == 'EMPTY']
+
+        container = None
+
+        for empty in empties:
+            if not empty.parent:
+                container = empty
+                break
+
         objects = []
+        if container:
+            # Children must be linked before parents,
+            # otherwise the hierarchy will break
+            nodes = list(container.children)
+
+            for obj in nodes:
+                obj.parent = asset_group
 
-        for obj in imported:
-            obj.parent = asset_group
+            bpy.data.objects.remove(container)
 
-        for obj in imported:
-            objects.append(obj)
-            imported.extend(list(obj.children))
+            for obj in nodes:
+                objects.append(obj)
+                objects.extend(list(obj.children_recursive))
```

Comment:
Is it important that `obj` children always come directly after the `obj` or can we first put in all `nodes` and then add all their children to the list?

E.g. would this work?
```suggestion
            objects.extend(nodes)
            for obj in nodes:
                objects.extend(obj.children_recursive)
```

Reply:
It is a remnant of when they were collections! It would mess up the hierarchy if you didn't link them in order. No longer necessary :)